cmd/main.go

```diff
@@ -56,14 +55,18 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
+	var (
+		runLocal             bool
+		metricsAddr          string
+		enableLeaderElection bool
+		probeAddr            string
+	)
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.BoolVar(&runLocal, "run-local", false, "Configures whether the CPOM will run in local dev mode.")
```

Comment:
Non-blocking nit: see https://github.com/openshift/platform-operators/pull/23#discussion_r965184103

Reply:
I ended up removing this functionality entirely. I think it's still something that's beneficial for QoL but I didn't want to expand the scope of this work.